### ISSUE TITLE
Output scaler in model

### ISSUE THREAD

**Issue by viktor-ktorvi** [viktor-ktorvi/thesis-proposal#7]
It'd be nice if the model outputed the physical features instead of the normalized ones. So the forward function would be:

```python
y_norm = self.model(x)
y_physical = self.output_scaler.inverse(y_norm)

return y_physical
```
This is prefered because the final layer weights of the model are gonna output something normalized so they're gonna be on the same order of magnitude and that's gonna be practically easier for the solver.

Then for the supervised case we'd have two options:
1.
```python
loss = (target_physical - y_physical)**2
```

or 

2.
```python
loss = (model.output_scaler(target_physical) - model.output_scaler(y_physical))**2
```

According to my calculations gradient_1 = gradient_2 * sigma^2, where sigma = y_physical_std.

I think gradient_2 is the more desirable option.

**Comment by viktor-ktorvi** [comment]
On another node, the output scaler as is won't work for multiple topologies cause the target vectors are of different sizes. The output scaler will be the same as the input scaler effectively.
